app/src/main/java/com/grarak/kerneladiutor/utils/kernel/sound/Sound.java

```diff
@@ -194,9 +209,13 @@ public static void setSpeakerGain(String value, Context context) {
                     value = String.valueOf(newGain + 256);
                     fauxRun(value + " " + value, SPEAKER_GAIN, SPEAKER_GAIN, context);
                 }
+                break;
```

Comment:
Looks good, I'll probably find some other annoyance later and just submit a big cleanup patch at a later point